plugin/tests/unit/sdk_v2/test_sdk_v2_config_template.py

```
@@ -0,0 +1,326 @@
+
+
+@pytest.mark.asyncio
+async def test_config_template_base_and_profiles_views() -> None:
+    cfg = core_config.PluginConfig(_CtxFull())
+    # PluginConfigBaseView and PluginConfigProfiles are aliases to PluginConfig
+    assert isinstance(cfg, core_config.PluginConfigBaseView)
+    assert isinstance(cfg, core_config.PluginConfigProfiles)
+
+    assert (await cfg.base_dump())["feature"]["enabled"] is True
+    assert (await cfg.get_bool("feature.enabled")) is False  # dev profile overrides base
+    with pytest.raises(Exception):
```

Ruff (0.15.6)

[warning] Do not assert blind exception: `Exception`

(B017)